lib/src/client.dart

```diff
@@ -1801,13 +1805,27 @@ class Client extends MatrixApi {
         await _handleRooms(leave, direction: direction);
       }
     }
+
+    CachedPresence? lowestLastActivePresence;
+
     for (final newPresence in sync.presence ?? []) {
       final cachedPresence = CachedPresence.fromMatrixEvent(newPresence);
       presences[newPresence.senderId] = cachedPresence;
       // ignore: deprecated_member_use_from_same_package
       onPresence.add(newPresence);
       onPresenceChanged.add(cachedPresence);
+
+      if (lowestLastActivePresence == null ||
+          cachedPresence.lastActiveTimestamp!
+              .isBefore(lowestLastActivePresence.lastActiveTimestamp!)) {
```

Comment:
`cachedPresence.lastActiveTimestamp` can be null and `lowestLastActivePresence.lastActiveTimestamp` can be null